### DISCUSSION TITLE
如何在文内图表开头的页面顶端空一行

### DISCUSSION THREAD

**Discussion by lyuPang** [sjtug/SJTUThesis#689] [提问] [answered]
被挑错了，如何才能在图片作为开头的页面顶端加一个空行呢

**Comment by skyzh** [comment, marked as answer]
空行问题没有在[本科生毕业设计论文的指导意见](https://github.com/sjtug/SJTUThesis/files/6505296/default.pdf)中提到，建议联系您的审核老师。目前 LaTeX 应该没有比较好的修改办法，可以考虑调整 figure 的位置，将图片重排到其他位置。参考 https://liam.page/2017/04/30/floats-in-LaTeX-the-positioning-algorithm/ 的浮动体设置。

**Comment by skyzh** [comment]
也可以在出问题的 `figure` 环境开头加一个 `\bigskip` 作为 workaround。
